Runtime/TwitchManager.cs

```diff
@@ -59,6 +59,8 @@ public class TwitchManager : MonoBehaviour {
             ClientID = "",
             ChannelName = "",
             BotName = "",
+            PostConnectMessage = true,

```

Comment:
I would be inclined to default this to `false` so it doesn't unexpectedly appear for existing users of Twitchmata when they upgrade